src/app/components/home/components/ip-reports/ip-reports.component.html

```diff
@@ -0,0 +1,43 @@
+<div *ngIf="isLoaded" [@fadeInOut]="isLoaded" class="py-4 px-6 bg-surface-medium rounded-lg">
+  @if (!error.val) {
+    <div>
+      <h1 class="font-medium text-xl">Denúnicias</h1>
+      <p>{{ summary.reports.length }} resultados</p>
+    </div>
+    <br />
+    @if (summary.reports && summary.reports.length > 0) {
+      <mat-list>
+        @for (report of summary.reports; let i = $index; track i) {
+          <mat-list-item>
+            <span matListItemTitle
+              ><b>Denúncia {{ i + 1 }}</b></span
+            >
+            <div class="float-left max-w-[30%]">
+              <span matListItemLine><b>Relator</b>: {{ report.reporterId }}</span>
+              <span matListItemLine><b>Data</b>: {{ getDate(report.reportedAt) }}</span>
+            </div>
+            <div class="float-right min-w-[60%]">
+              <span matListItemLine><b>Comentário</b>: {{ report.comment }}</span>
+              <span matListItemLine><b>Categorias</b>: {{ report.categories.join(', ') }}</span>
+            </div>
```

Comment:
Tem alguns textos cortando quando são longos. O que acha de usar estes estilos?

```suggestion
            <span matListItemLine><b>Relator</b>: {{ report.reporterId }}</span>
            <span matListItemLine><b>Data</b>: {{ getDate(report.reportedAt) }}</span>
            <span matListItemLine class="max-w-full !whitespace-normal !break-all">
              <b>Comentário</b>:
              {{ report.comment }}
            </span>
            <span matListItemLine><b>Categorias</b>: {{ report.categories.join(', ') }}</span>
```

Reply:
Antes:
![Screenshot from 2024-09-16 11-02-31](https://github.com/user-attachments/assets/03dd127e-455b-4a1e-9291-a514e24868b0)

Depois:
![Screenshot from 2024-09-16 11-02-05](https://github.com/user-attachments/assets/b94e64cd-c655-4518-97f3-cfd8ba29f4e9)